Kconfig

```diff
@@ -0,0 +1,16 @@
+menuconfig TFM
```

Comment:
@oyvindronningstad could this be moved to the Zephyr side instead of the module?
E.g. in modules/...../tfm, or, maybe a subsystem if module directory is not good enough

Reply:
Please move this into the main zephyr tree

Reply:
Done